opm/output/eclipse/Summary.cpp

```diff
@@ -853,6 +861,13 @@ void Summary::add_timestep( int report_step,
 
     const size_t timestep = report_step;
     const auto& schedule = es.getSchedule();
+    const std::vector<double>& porv_all = es.get3DProperties().getDoubleGridProperty("PORV").getData();
+    const size_t  num = this->grid.getNumActive();
+    std::vector<double> porv (num,0.0);
+    for (size_t active_index = 0; active_index < num ; ++active_index) {
+        const size_t global_index = this->grid.getGlobalIndex(active_index);
```

Comment:
A bit excessive to make this compressed copy for every single timestep - a `porv` property should be added as a private member to the `Summary` class - and initialized in the constructor.